install/bin/decj

```diff
@@ -25,16 +35,10 @@ fi
 # Make sure DCSTOOL_USERDIR is set
 if [ -z "$DCSTOOL_USERDIR" ]
 then
-  decj_owner=`ls -l "$0" | awk '{ print $3 }'`
-  if [ "$decj_owner" = "$USER" ]
-  then
-    DCSTOOL_USERDIR=$DCSTOOL_HOME
-  elif [ -d $HOME/opendcs_user ]
-  then
-    DCSTOOL_USERDIR=$HOME/opendcs_user
-  else
-    DCSTOOL_USERDIR=$HOME
-  fi
+  DCSTOOL_USERDIR=$HOME/.opendcs
+  echo "Creating Local User Directory and initial properties in ${DCSTOOL_USERDIR}"
+  mkdir -p $DCSTOOL_USERDIR
+  cp $DH/decodes.properties $DCSTOOL_USERDIR/user.properties
```

Comment:
This should probably be part of install ;  If this runs every time that may clobber data.
May need to update documentation.

Reply:
OH, I see this only runs if DCSTOOL_USERDIR doesn't exist.

Reply:
Though I do need to check that the folder doesn't exist already. I think I forgot that part.

Reply:
Also didn't want to put it as part of the install as we may be getting away from the installer.